packages/core/test/toast/overlayToasterTests.tsx

```diff
@@ -108,13 +108,31 @@ describe("OverlayToaster", () => {
                 });
             });
 
-            it("multiple show()s renders them all", () => {
+            it("multiple show()s renders them all", async () => {
                 toaster.show({ message: "one" });
                 toaster.show({ message: "two" });
                 toaster.show({ message: "six" });
+                await delay(150);
```

Comment:
Is there any way we can tie these hardcoded values to the value of `QUEUE_TIMEOUT_MS`? That might make them a bit less ambiguous in the context of these tests. Perhaps we could leave a comment or even export reference the constant itself in code:
```ts
await delay(3 * QUEUE_TIMEOUT_MS);
```

Also, should we be using [timer mocks](https://sinonjs.org/releases/latest/fake-timers/) in our tests?

Reply:
Yeah good call on the delay, will export that. I attempted to user the timer mocks and I wasn't able to get them to work, will give them another shop.

Reply:
Figured out what I messed up and switched to fake timers 👍